spec/Overview.html

```diff
@@ -2061,10 +2060,7 @@ <h4>The deriveBits method</h4>
               </li>
               <li>
                 <p>
-                  Let |result| be a new {{ArrayBuffer}}
-                  associated with the
-                  [= relevant global object =]
-                  of `this` [[HTML]], and
+                  Let |result| be the result of [= ArrayBuffer/create | creating =] an {{ArrayBuffer}}
                   containing the result of performing the derive bits operation
```

Comment:
I would have imagined the “containing” part to be written as [“with”](https://infra.spec.whatwg.org/#algorithm-params) as the [byte sequence](https://infra.spec.whatwg.org/#byte-sequences) is an argument for the create algorithm (along with the realm argument, though iiuc that’s being treated as an implicit here). However I’m not an expert on this and couldn’t say for sure whether this phrasing already fits expectations.